__tests__/app.test.js

```diff
@@ -446,4 +511,47 @@ describe('Error-handling Sad paths', () => {
             });
         });
     });
+    describe('Invalid query values', () => {
+        describe('GET /api/articles', () => {
+            describe('topic', () => {
+                it('should return a 404 not found if queried with a non-existent topic value', () => {
+                    return request(app)
+                        .get("/api/articles?topic=notATopic")
+                        .expect(404)
+                        .then(({ body }) => {
+                            expect(body).toStrictEqual({
+                                message: "Not Found",
+                                status: 404
+                            })
+                        })
+                });
+            });
+            describe('sort_by', () => {
+                it('should return a 400 bad request if queried with a non-existent column to sort by', () => {
+                    return request(app)
+                    .get("/api/articles?sort_by=notAColumn")
+                    .expect(400)
+                    .then(({ body }) => {

```

Comment:
You can also consider whether a 422: unprocessable entity would be a suitable status code 
(personal choice, I think a 400 is acceptable here too)